subsys/net/lib/config/init.c

```diff
@@ -401,7 +401,7 @@ int net_config_init(const char *app_info, uint32_t flags, int32_t timeout)
 	/* Loop here until we are ready to continue. As we might need
 	 * to wait multiple events, sleep smaller amounts of data.
 	 */
-	while (count--) {
```

Comment:
Can you rewrite your commit message without line numbers?

Reply:
I will first check, whether it is still required after
https://github.com/zephyrproject-rtos/zephyr/pull/25611#issuecomment-635179219.

Reply:
It is still required and modified to match changes on master branch.